onnxruntime/core/providers/shared_library/provider_interfaces.h

```diff
@@ -1580,7 +1604,6 @@ struct Tensor final {
   bool IsDataType() const;
 
   Tensor() = delete;
-  Tensor(const Tensor&) = delete;
```

Comment:
Why was this removed? You can't copy construct tensors by value

Reply:
Recovered, thanks!